content/docs/manage/branches.md

```diff
@@ -57,7 +57,23 @@ To view the branches in a Neon project:
 1. In the Neon Console, select a project.
 2. Select **Branches** to view the branches for the project.
 3. Select a branch from the table to view details about the branch.
-![Branch details](/docs/manage/branch_details.png)
+
+Branch details shown on the branch page include:
+
+- **ID**: The branch ID. Branch IDs have a `br-` prefix.
+- **Created**: The date and time the branch was created.
+- **Current Data Size**: The current data size of the branch.
+- **Active Time**: The amount of active time for the current month for the compute endpoint associated with the branch.
+- **Compute Time**: The amount of compute time 
```

Comment:
OCD kicked in. This doesn't have a dot at the end.

Reply:
Thank you! I have fixed this and I aligned the descriptions with those used on the Usage widget too tips.